app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<!--
+  <% @user.products.each do |product| %>
+    <p><%= link_to product.name, product_path(product.id) %></p>
+    <p><%= image_tag(product.photo_url)%></p>
+    <p>In Stock: <%= product.stock %></p>
+    <p>Price: $<%= product.price %></p>
+  <% end %> -->
+
+
+<h1><%= @user.username  %></h1>
+<div class="row">
+<% @user.products.where(status: "active").each do |product| %>
```

Comment:
I think I saw this line somewhere else as well - this logic should not be in the view